developer/ckcp/openshift-operators/minio/tenant/tenant.yaml

```diff
@@ -0,0 +1,69 @@
+
+apiVersion: minio.min.io/v2
+kind: Tenant
+metadata:
+  name: storage
+  namespace: tekton-results
+  labels:
+    app: minio
+  annotations:
+    prometheus.io/path: /minio/v2/metrics/cluster
+    prometheus.io/port: "9000"
+    prometheus.io/scrape: "true"
+spec:
+  exposeServices:
+    console: true
+    minio: true
+  features:
+    bucketDNS: false
+    domains: { }
+  # It doesn't work. TODO: create an issue for minio operator
+  # users:
+  #   - name: storage-user
+  # buckets:
+  #   - name: "tekton-results"
+  #     region: "us-east-1"
+  #     objectLock: true
```

Comment:
Can you describe what the issue is? Or do you plan to submit an issue upstream?

Reply:
I submitted: https://github.com/minio/operator/issues/1345 and https://github.com/minio/operator/issues/1346